src/styles/index.scss

```diff
@@ -1,3 +1,132 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  width: 200px;
+  height: 408px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.product-card__block-img {
+  padding: 32px 19px 0;
+}
+
+.product-card__name {
+  padding: 36px 16px 4px;
+}
+
```

Comment:
Need to use scss nesting everywhere
<img width="594" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/abed1b5f-f7dd-4394-822f-6763639b8018">